### DISCUSSION TITLE
ios will not open GitHub links in GitHub

### DISCUSSION THREAD

**Discussion by fulldecent** [community/community#27979] [Mobile]
In the past year, links from Gmail and Safari to the GitHub domain stopped opening in the GitHub iOS app. 

This makes the app basically useless. Since it's too hard to get from an email notification to commenting in an issue. 

I'm not sure how it got messed up. But I hope there can be a button  it the iOS GitHub app to reset or reregister the URL support.

**Comment by RoderickJDunn** [comment]
Delete/reinstall of the GitHub app fixed it for me

**Reply by user454322** [reply]
Reinstalling didn't work for me but in the new version I can change the icon of the app

**Reply by rakira2309** [reply]
Ued

**Reply by jcshort** [reply]
I've found that it only works _once_ each time the app is re-installed.

**Comment by mcmxcdev** [comment]
> Delete/reinstall of the GitHub app fixed it for me

Doesn't work for me with Proton Mail, just reinstalled GitHub app, no luck.

**Reply by Ppoollkkmmnn09** [reply]
Cant parse page

Can mercene me video how to fix it

**Comment by fedorkanan** [comment]
hi dear William Entriken <3
The section headings should not contain any special characters. And the link referencing said section should be hyphen separated and in lowercase.

[This is an example] (#section-heading)

**Comment by cagnulein** [comment]
Same here, any news?

**Comment by Gogo200202** [comment]
I don't know solution for ios but for some android have same issue and i use this [URLCheck](https://f-droid.org/packages/com.trianguloy.urlchecker/)

**Comment by 0xvasanth** [comment]
Reinstall solves for me in IOS.

**Reply by lev-tonkean** [reply]
I just tried reinstalling, loging out and logining in. In both the app and safari. Nothing helps...

**Reply by richtera** [reply]
I tried reinstall several times. This used to work, but doesn't work anymore for me.

**Comment by akashprakash12** [comment]
Go to Settings > GitHub App > Default Browser App ,Set it to your preference to ensure GitHub links open in the correct place.

**Reply by richtera** [reply]
There is no such setting though (at least not currently on IOS)

**Comment by tony9823** [comment]
Why did links from Gmail and Safari to the GitHub domain stop opening in the GitHub iOS app? This issue makes it difficult to navigate from email notifications to commenting on issues. [here]( https://hazecbd.co.uk/cbd-vape-juice/) the app include a button to fix this?

**Comment by Ppoollkkmmnn09** [comment]
> In the past year, links from Gmail and Safari to the GitHub domain stopped opening in the GitHub iOS app. 
> 
> This makes the app basically useless. Since it's too hard to get from an email notification to commenting in an issue. 
> 
> I'm not sure how it got messed up. But I hope there can be a button  it the iOS GitHub app to reset or reregister the URL support.


Cant parse page
Can mercene me video how to fix it
![Screenshot_٢٠٢٥٠١٢١-١٩٣٦٥٦.jpg](https://github.com/user-attachments/assets/f564999f-493d-448c-bce2-bdb86e97ff88)

 😡
